src/tribol/utils/Math.cpp

```diff
@@ -120,10 +120,7 @@ void bubble_sort( int* const array, const int n )
 }
 
 //------------------------------------------------------------------------------
-RealT abs_val_diff( RealT val1, RealT val2 )
-{
-  return std::abs(val1 - val2);
-}
+RealT abs_val_diff( RealT val1, RealT val2 ) { return std::abs( val1 - val2 ); }
```

Comment:
Is this still used?  Can remove if not.

Reply:
It is used in some tests. Cleaned things up a little. I'll just leave it.